src/convertToObject.js

```diff
@@ -12,7 +12,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceConverted = {};
+
+  sourceString.split(';').forEach(rule => {
+    const splitedRule = rule.trim().split(':');
```

Comment:
Ось цей .trim() якщо забрати то має так само працювати! Не пробував? Ти ж 2 частини цього ж  рядка і так пізніше чистиш від пробілів..
І дякую над своїм кодом теж подумаю)
```suggestion
    const splitedRule = rule.split(':');
```